pygmt/tests/test_text.py

```diff
@@ -371,3 +371,16 @@ def test_text_nonstr_text():
         text=[1, 2, 3.0, 4.0],
     )
     return fig
+
+
+@pytest.mark.mpl_image_compare
+def test_text_nonascii():
+    """
+    Test passing text strings with non-ascii characters.
+    """
+    fig = Figure()
+    fig.basemap(region=[0, 10, 0, 10], projection="X10c", frame=True)
+    fig.text(position="TL", text="position-text:°α")
+    fig.text(x=1, y=1, text="xytext:°α")
+    fig.text(x=[5, 5], y=[3, 5], text=["xytext1:°α", "xytext2:°α"])
```

Comment:
Mix in some Symbol and ZapfDingbats characters too from https://docs.generic-mapping-tools.org/latest/cookbook/octal-codes.html#octal-codes-symbol-zap?

Reply:
Done in adf6eda9d4f88de10ec4169e6f662d0796a48e73.